src/main/java/core/basesyntax/FileEditor.java

```diff
@@ -0,0 +1,188 @@
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Date;
+import java.util.Scanner;
+
+public class FileEditor {
+    private static final String CREATE = "create";
+    private static final String READ = "read";
+    private static final String INFO = "info";
+    private static final String HELP = "help";
+    private static final String EXIT = "exit";
+    private static final String ENTER_COMMAND = "enter command";
+    private static final String WRONG_PARAMETERS_FOR_COMMAND = "wrong parameters for command %s";
+    private static final String UNKNOWN_COMMAND = "%s - unknown command";
+    private static final String WANT_SAVE = "do you want to save it in file (y/n)?";
+    private static final String ENTER_PATH_END_FILENAME
+            = "enter path and filename trough the space:";
+    private static final String INCORRECT_PATH_OR_FILENAME = "%s - incorrect path and filename";
+    private static final String FILE_NOT_FOUND = "file not found";
+    private Scanner scanner;
+    private String inputString;
+
+    public FileEditor() {
+        scanner = new Scanner(System.in);
+    }
+
+    public void run() {
+        while (true) {
+            System.out.println(ENTER_COMMAND);
+            inputString = scanner.nextLine();
+            String[] args = inputString.replaceAll("\\s+", " ").trim().split(" ");
+            if (args.length > 0) {
+                handleCommands(args);
+            }
+        }
+    }
+
+    private void handleCommands(String[] args) {
+        switch (args[0]) {
+            case EXIT:
+                if (args.length == 1) {
+                    System.exit(0);
+                } else {
+                    System.out.println();
+                }
+                break;
+            case CREATE:
+                if (args.length == 3) {
+                    createFile(args[1], args[2], "");
+                } else {
+                    System.out.println(String.format(WRONG_PARAMETERS_FOR_COMMAND, args[0]));
+                }
+                break;
+            case READ:
+                if (args.length == 3) {
+                    readFile(args[1], args[2]);
+                } else {
+                    System.out.println(String.format(WRONG_PARAMETERS_FOR_COMMAND, args[0]));
+                }
+                break;
+            case INFO:
+                if (args.length == 3) {
+                    readFileInfo(args[1], args[2]);
+                } else {
+                    System.out.println(String.format(WRONG_PARAMETERS_FOR_COMMAND, args[0]));
+                }
+                break;
+            case HELP:
+                if (args.length == 2) {
+                    try {
+                        Command cmd = Command.valueOf(args[1].toUpperCase());
+                        for (String help : cmd.getHelp()) {
+                            System.out.println(cmd.toString() + " - " + help);
+                        }
```

Comment:
we can simplify that if cmd.getHelp() would return just String but not array

Reply:
done